README.md

```diff
@@ -100,7 +100,7 @@ You can still download the latest, though. They are cool!
 Download the latest [brain.js for browser](https://raw.githubusercontent.com/harthur-org/brain.js/master/browser.js). Training is computationally expensive, so you should try to train the network offline (or on a Worker) and use the `toFunction()` or `toJSON()` options to plug the pre-trained network into your website.
 
 # Training
-Use `train()` to train the network with an array of training data. The network has to be trained with all the data in bulk in one call to `train()`. The more training patterns, the longer it will probably take to train, but the better the network will be at classifying new patterns.
+Use `train()` to train the network with an array of training data. The network has to be trained with all the data in bulk in one call to `train()`. The more training patterns, the longer it probably takes to train, and the better the network will be at classifying new patterns.
```

Comment:
```The more training patterns, the longer it probably takes to train, and the better the network will be at classifying new patterns.```

Kinda sounds funny.  What do you think of this (or something like it)?

```More training patterns will probably take longer to train, but will usually result in a network better at classifying new patterns.```

Reply:
I think the latter one is better.